lib/watcher/appmetrics.js

```diff
@@ -0,0 +1,53 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
```

Comment:
can you add a comment explaining what this is for? that it produces events used by the Liberty collective? so no one deletes it in the future on the assumption that is unused :-)